File: app/dbmodels/UsedSoftwareLicense.php
<?php

$table = array(
    'fields'        => array(),
    'oldfields'     => array(),
    'renamefields'  => array(),
    'indexes'       => array(),
    'oldindexes'    => array(),
    'relationships' => array(),
    'model'         => 'UsedSoftwareLicense',
    'menu'          => ''
);

$table['fields'] = array(
    'id'                  => DBModels::type('increments', 
                                            array('visible' => false)),
    'asset_id'            => DBModels::type('integer'),
    'software_license_id' => DBModels::type('integer'),
    'is_deleted'          => DBModels::type('boolean'),
    'is_dynamic'          => DBModels::type('boolean'),
);

File: fusionglpi/database/used_software_license.py
from fusionglpi import db

class UsedSoftwareLicense(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    asset_id = db.Column(db.Integer, db.ForeignKey('asset.id'))
    software_license_id = db.Column(db.Integer, db.ForeignKey('software_license.id'))
    is_deleted = db.Column(db.Boolean, nullable=False, server_default="0")
    is_dynamic = db.Column(db.Boolean, nullable=False, server_default="0")
